test/desdemona/query_test.clj

```diff
@@ -132,16 +166,26 @@
        [{:ip "10.0.0.2"    ;; ip clause succeeded
          :type "ingress"}]])))
 
-(deftest logic-query-tests
+(deftest run-logic-query-tests
   (are [query results] (= results (#'q/run-logic-query query events))
     'l/fail
     []
 
-    '(l/featurec x {:ip "10.0.0.1"})
+    (list clojure.core.logic/featurec
+          (#'q/free-sym 'x)
+          {:ip "10.0.0.1"})
+    [[{:ip "10.0.0.1"}]]
+
+    (list clojure.core.logic/featurec
+          (#'q/free-sym 'y)
+          {:ip "10.0.0.1"})
```

Comment:
(It might be reasonable to rely on dsl->logic here; I appreciate the desire to test exactly one thing per test, but we do test that unit separately, and for any reasonably complex query this is going to turn into an eyesore...)